src/witan/send/adroddiad/valid_states.clj

```diff
@@ -20,22 +20,24 @@
     (ay/key-stage-3 y) (apply max ay/key-stage-3)
     (ay/key-stage-4 y) (apply max ay/key-stage-4)
     (ay/key-stage-5 y) (apply max ay/key-stage-5)
-    (ay/ncy-15+ y) (apply max ay/ncy-15+)))
+    (ay/ncy-15+ y) (apply max ay/ncy-15+)
+    (< (apply max ay/ncy-15+) y) (apply max ay/ncy-15+)))
```

Comment:
I like this one and I presume it will blow up otherwise. I suppose it isn't terrible if this blows up b/c there is a stray 99 in the data though, which makes me wonder if we should be throwing an ex-info instead as I think the assumption should be that the data is clean by this point.